app/scripts/services/validateRegistrant.js

```diff
@@ -170,6 +170,12 @@ angular.module('confRegistrationWebApp')
               return;
             }
             break;
+          case 'checkboxQuestion':
+            if (_.isEmpty(answer) || _.isEmpty(_.pickBy(answer, function(val){ return val; }))) {
```

Comment:
The default predicate for `_.pickBy` is Lodash's `_.identity` function so you should be able to use it without the second parameter to keep truthy values.
```suggestion
            if (_.isEmpty(answer) || _.isEmpty(_.pickBy(answer)) {
```